src/components/Logo/AssetLogo.tsx

```diff
@@ -0,0 +1,60 @@
+import { SupportedChainId } from 'constants/chains'
+import useTokenLogoSource from 'hooks/useAssetLogoSource'
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { MissingImageLogo } from './BaseLogo'
+
+const LogoImage = styled.img<{ size: string }>`
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  background: radial-gradient(white 60%, #ffffff00 calc(70% + 1px));
```

Comment:
Did you intentionally change the radial-gradient? It used to be

    radial-gradient(white 50%, #ffffff00 calc(75% + 1px), #ffffff00 100%)

Reply:
Yeah, I asked design if we still wanted the shadow gradient inside of transparent images, they said no take it out, but then I realized if I made the background just white, there was bleed-through for some mostly-opaque images, so instead I left the gradient, but made it tighter:
old:
<img width="420" alt="image" src="https://user-images.githubusercontent.com/39385577/202530536-2b1ce294-a635-4224-8435-52a7067f0fc8.png">

new:
<img width="411" alt="image" src="https://user-images.githubusercontent.com/39385577/202530481-ed9e8b9f-ef6d-4141-8232-25466949b2ce.png">